src/course-outline/drag-helper/DraggableList.tsx

```diff
@@ -33,33 +74,34 @@ const DraggableList = ({
   handleSubsectionDragAndDrop,
   handleUnitDragAndDrop,
   children,
-}) => {
-  const prevContainerInfo = React.useRef();
+}: DraggableListProps) => {
+  const prevContainerInfo = React.useRef<string | null>();
   const sensors = useSensors(
     useSensor(PointerSensor),
     useSensor(KeyboardSensor, {
       coordinateGetter: sortableKeyboardCoordinates,
     }),
   );
-  const [activeId, setActiveId] = React.useState();
-  const [currentOverId, setCurrentOverId] = React.useState();
+  const [activeId, setActiveId] = React.useState<UniqueIdentifier | null>();
```

Comment:
We can use the following if we prefer `undefined`
```suggestion
  const [activeId, setActiveId] = React.useState<UniqueIdentifier>();
```
or the following if we prefer null:
```suggestion
  const [activeId, setActiveId] = React.useState<UniqueIdentifier>(null);
```

Reply:
Fixed it by setting it like below:

```tsx
  const [activeId, setActiveId] = React.useState<UniqueIdentifier | null>(null);
```